docs/draft_production_docs/oadp-self-service.md

```diff
@@ -0,0 +1,484 @@
+
+A non-admin user can create a NABSL in their namespace.
+
+```
+apiVersion: oadp.openshift.io/v1alpha1
+kind: NonAdminBackupStorageLocation
+metadata:
+  name: nacuser1-nabsl
+  namespace: nacuser1
+spec: 
+  backupStorageLocationSpec:
+    config:
+    checksumAlgorithm: ""
+    profile: default
+    region: us-west-2
+    credential:
+      key: cloud
+      name: cloud-credentials
+    objectStorage:
+      bucket: bucket1uswest2
+      prefix: velero
+    provider: aws  
+```
+
+If the cluster administrator has enabled the requireApprovalForBSL flag then the NABSL will be in the Pending state until the cluster administrator approves the NABSL.
+
+
+
+### NAB / NAR Status
+
+#### Phase
+The phase field is a simple one high-level summary of the lifecycle of the objects, that only moves forward. Once a phase changes, it can not return to the previous value.
+
+| **Value** | **Description** |
+|-----------|-----------------|
+| New | *NonAdminBackup/NonAdminRestore* resource was accepted by the NAB/NAR Controller, but it has not yet been validated by the NAB/NAR Controller |
+| BackingOff | *NonAdminBackup/NonAdminRestore* resource was invalidated by the NAB/NAR Controller, due to invalid Spec. NAB/NAR Controller will not reconcile the object further, until user updates it |
+| Created | *NonAdminBackup/NonAdminRestore* resource was validated by the NAB/NAR Controller and Velero *Backup/restore* was created. The Phase will not have additional information about the *Backup/Restore* run |
+| Deletion | *NonAdminBackup/NonAdminRestore* resource has been marked for deletion. The NAB/NAR Controller will delete the corresponding Velero *Backup/Restore* if it exists. Once this deletion completes, the *NonAdminBackup/NonAdminRestore* object itself will also be removed |
+
+
+
+
+
+## Advanced Cluster Administrator Features
+
+### Cluster Administrator Enforceable Spec Fields
+There are several types of cluster scoped objects that non-admin users should not have access to backup or restore.  OADP self-service automatically excludes the following list of cluster scoped resources from being backed up or restored.
+
+* SCCs
+* ClusterRoles
+* ClusterRoleBindings
+* CRDs
+* PriorityClasses
+* virtualmachineclusterinstancetypes
+* virtualmachineclusterpreferences
+
+Cluster administrators may also enforce company or compliance policy by utilizing templated NABSL's, NAB's and NAR's that require fields values to be set and conform to the administrator defined policy.  Admin Enforceable fields are fields that the cluster administrator can enforce non cluster admin users to use. Restricted fields are automatically managed by OADP and cannot be modified by either administrators or users.
+
+#### NABSL
+The following NABSL fields are currently supported for template enforcement:
+
+| **NABSL Field**            | **Admin Enforceable** | **Restricted** | **special case** |
+|----------------------------|-----------------|----------------|-----------------|
```

Comment:
copy the updated tables from https://github.com/migtools/oadp-non-admin/issues/151

Reply:
+1 we're missing a couple things but THANK YOU!